schemas/file.yml

```diff
@@ -21,13 +21,33 @@
   - name: directory
     level: extended
     type: keyword
-    description: Directory where the file is located.
+    short: Directory where the file is located.
+    description: >
+      Directory where the file is located.
+
+      On platforms with drive letters, the drive letter should not be part of
+      the directory field. For example, "C:\\Program Files\\Foo"
+      should be parsed as `file.directory` "Program Files\\Foo" and
+      `drive.letter` "C".
     example: '/home/alice'
 
+  - name: drive
+    level: extended
+    type: keyword
+    description: Drive letter where the file is located.
+    example: C
```

Comment:
@rw-access @andrewkroh Any preference on whether this should be `C:` or `C`?

I'm currently proposing stripping off the colon because it doesn't carry any information. But if people are used to seeing the `:` as part of their "drive letter", that's fine too. It's been so long since I used Windows seriously that I have no recollection what the convention is, for that 😬